packages/block-editor/src/hooks/color.js

```diff
@@ -139,6 +139,9 @@ const ColorPanel = ( { colorSettings, clientId } ) => {
 
 	useEffect( () => {
 		const colorsDetectionElement = getBlockDOMNode( clientId );
+		if ( ! colorsDetectionElement ) {
```

Comment:
This is a small unrelated bug fix that was causing some blocks (template part) to fail in the edit site page.